src/lib/fetcher.ts

```diff
@@ -23,12 +24,24 @@ export default class Fetcher implements APIFetcher {
       return []
     }
 
-    const response = await this.makeRequest({
-      method: 'GET',
-      url: `/entries?sys.contentType.sys.id[in]=${ids.join(',')}`
-    })
+    let entries: APIEntry[] = []
+    let skip: number = 0
+
+    while (true) {
+      const response = await this.makeRequest({
+        method: 'GET',
+        url: `/entries?sys.contentType.sys.id[in]=${ids.join(',')}&skip=${skip}`
+      })
+
+      entries = entries.concat(response.items)
+      skip += 100
```

Comment:
I'd prefer not to have a magic number here and use `response.items.length`.

Reply:
Good point!